src/diffpy/srmise/pdfdataset.py

```diff
@@ -202,7 +202,10 @@ def read(self, filename):
         self
         """
         try:
-            self.readStr(open(filename, "rb").read())
+            # Open the file in binary mode, read it, and decode to convert bytes to string
+            with open(filename, "rb") as file:
+                file_content = file.read().decode("utf-8")
```

Comment:
Bug fix - in the latest Python, `rb` is rendered as a byte-object. Must explicitly convert to a text so that we can parse the file.